test_rails_4_app/spec/rake_task_spec.rb

```diff
@@ -1,18 +1,14 @@
 require 'rails_helper'
+require 'rake'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.